resources/shaders/deferred_composition.hlsl

```diff
@@ -84,7 +85,8 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 			is_hybrid);
 
 		// Shade pixel
-		retval = shade_pixel(pos, V, albedo, metallic, roughness, normal, sampled_irradiance, reflection);
+		float3 occluded_irradiance = sampled_irradiance * gbuffer_AO[screen_coord].x;
+		retval = shade_pixel(pos, V, albedo, metallic, roughness, normal, occluded_irradiance, reflection);
```

Comment:
Use lerp instead (use sampled_arradiance) if it's not hybrid

Reply:
fixed!